src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -1396,4 +1401,103 @@ describe('NewWorkspaceModal', () => {
       })
     );
   });
+
+  describe('shows egress warnings for cloning GCP workspaces', () => {
+    it('shows a message if the destination bucket location is in a different region', async () => {
```

Comment:
This is a test case for our existing functionality (which was written before we had much of anything in the way of unit tests).